data/src/main/java/org/oppia/android/data/backends/gae/NetworkModule.kt

```diff
@@ -20,7 +20,7 @@ import javax.inject.Singleton
 class NetworkModule {
 
   @Qualifier
-  private annotation class OppiaRetrofit
+  annotation class OppiaRetrofit
```

Comment:
Prefer moving this to its own file rather than importing the module elsewhere--modules should only be imported when defining components.

Reply:
Moved to a separate file.